src/tests.ts

```diff
@@ -4,6 +4,8 @@ import Adapter from "@wojtekmaj/enzyme-adapter-react-17"
 import "regenerator-runtime/runtime"
 import { format } from "util"
 import "@testing-library/jest-dom"
+// this import makes sure that the translations are accessible in the test environment
+import "System/i18n/i18n"
```

Comment:
🙏

Reply:
Thanks for the solution! Can you check if we had previous tests mocking it that don't need it anymore?